core/src/main/scala/io/finch/EndpointResult.scala

```diff
@@ -1,8 +1,9 @@
 package io.finch
 
+import cats.effect.Effect
 import com.twitter.finagle.http.Method
-import com.twitter.util.{Await, Duration, Future, Try}
-import io.catbird.util.Rerunnable
+import com.twitter.util._
+import scala.concurrent.duration.{Duration => ScalaDuration}
```

Comment:
It looks like the next item on our agenda should be migrate off Twitter's `Duration` to Scala `Duration`.

Reply:
Agree, it's confusing

Reply:
Let's create a ticket so we don't forget.